apps/brandfolder/src/index.js

```diff
@@ -0,0 +1,110 @@
+import pick from 'lodash/pick';
+import { setup } from 'shared-dam-app';
+
+const CTF_APP_URL = 'https://brandfolder.ctfapps.net';
+const BF_EMBED_URL = `https://integration-panel-ui.brandfolder-svc.com?channel=message&appName=Contentful&origin=${CTF_APP_URL}&initMsg=hi`;
+
+const CTA = 'Select an asset on Brandfolder';
+
+const FIELDS_TO_PERSIST = [
+  'id',
+  'type',
+  'mimetype',
+  'extension',
+  'filename',
+  'size',
+  'width',
+  'height',
+  'url',
+  'thumbnail_url',
+  'position',
+  'relationships',
+  'included',
+  'asset.id'
```

Comment:
'cdn_url' should also be allowed. Please add that, or I can create a PR for it.

Reply:
I've added it here:
https://github.com/contentful/apps/pull/62